src/EFTools/SharedAssemblyInfo.cs

```diff
@@ -24,7 +24,7 @@
 #elif VS15
 [assembly: AssemblyVersion("15.0.0.0")]
 [assembly: AssemblyFileVersion("15.0.0.0")]
-#elif VS16
+#elif VS16 || VS17
```

Comment:
It seems the targets set the preprocessor directive based on the VS version, and the reason we're shipping a version 16 is because we're still building with a VS2019 toolchain. That's probably not what we want, but this at least codifies that.